src/bondTokenLogoUriDedupeMap.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
We've discussed how we can eliminate the need for this, but it's not super important.